op-mode-definitions/show-log.xml.in

```diff
@@ -212,6 +212,31 @@
               </tagNode>
             </children>
           </node>
+          <node name="protocol">
+            <properties>
+              <help>Show log for Routing Protocols</help>
+            </properties>
+            <children>
+              <leafNode name="ospf">
+                <properties>
+                  <help>Show log for OSPF</help>
+                </properties>
+                <command>journalctl -b /usr/lib/frr/ospfd</command>
+              </leafNode>
+              <leafNode name="ospfv6">
```

Comment:
Should it be **ospfv3**?

```
vyos@r1# set protocols ospf
Possible completions:
 > ospf         Open Shortest Path First protocol (OSPF) parameters
 > ospfv3       IPv6 Open Shortest Path First protocol (OSPFv3) parameters
```

Reply:
I have corrected the ospfv6 to ospfv3 and also added the rest of the protocols for logging